### PR TITLE
Checklist status sync issue - CPLP2247

### PR DESCRIPTION
**Pull request by manojava-gk**
Approve/Decline/Retrigger/Cancel Process actions should clear button and update the latest status in both overlay and application list

### CODE REVIEW

File: cx-portal/src/features/admin/registration/actions.ts [hunk 1]
```diff
@@ -80,10 +80,22 @@ const declineRequest = createAsyncThunk(
...
 
+const refreshApplicationRequest = createAction(
+  `${name}/refreshApplicationRequest`,
+  function update(refresh: number) {
+    return {
+      payload: {
+        refresh,
+      },
+    }
+  }
```
Comment (oyo) [inline cx-portal/src/features/admin/registration/actions.ts:91]: Does that function need a name? I guess we can write less verbose as
`(refresh: number) => ({ payload: { refresh } })`

**Review by oyo — CHANGES_REQUESTED** [review]
small change

File: cx-portal/src/features/admin/registration/actions.ts (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/features/admin/registration/actions.ts:91]: I have just used as earlier. Will update it

**Review by oyo — APPROVED** [review]
looking good now

**Pull request merged**